### ISSUE TITLE
After Effects: The shortcut property mask of an AVLayer is not of type MaskPropertyGroup.

### ISSUE THREAD

**Issue by RaymondClr** [docsforadobe/Types-for-Adobe#138]
The mask property, which has the match name ADBE Mask Parade, has a propertyType of PropertyType.INDEXED_GROUP. It is expected to be a PropertyGroup, while its child properties are of the type MaskPropertyGroup.

For example: layer.mask.maskMode provides the correct type hint, but when executing the code, it fails to retrieve any values.

I used a third-party tool to demonstrate this hierarchy issue. This tool can read nested Property hierarchies and display them as JSON:
![image](https://github.com/user-attachments/assets/2ea14dab-fe5d-4d1c-ac22-7212412cb26b)

**Comment by zlovatt** [comment]
Yes, the current typedefs don't have any way to identify results accessed by matchnames or strings.

You can either check whether it's a MaskPropertyGroup after the fact, or access it from the layer via `avLayer.mask` (instead of `avLayer.property("ADBE Mask Parade")` to return the typed `MaskPropertyGroup` group.
